packages/manager/modules/web-universe-components/src/order/order.html

```diff
@@ -0,0 +1,102 @@
+<oui-stepper
+    data-current-index="$ctrl.currentIndex"
+    data-on-finish="$ctrl.validateCheckout()"
+>
+    <div data-ng-transclude></div>
+
+    <oui-step-form
+        data-header="{{:: 'wuc_order_pricing_header' | translate }}"
+        data-editable="$ctrl.pricings.length > 1 && !$ctrl.loading.getCheckoutInformations && !$ctrl.loading.validateCheckout"
+        data-valid="$ctrl.pricing"
+        data-prevent-next="true"
+        data-on-focus="$ctrl.getPricings()"
+    >
+        <div class="mb-3" data-ng-if="$ctrl.hasUniquePricing()">
+            <span
+                data-ng-pluralize
+                data-count="$ctrl.pricings[0].interval"
+                data-when="{
+                  'one': ('wuc_order_pricing_' + $ctrl.pricings[0].intervalUnit | translate:{ number: $ctrl.pricings[0].interval }),
+                  'other': ('wuc_order_pricing_' + $ctrl.pricings[0].intervalUnit + 's' | translate:{ number: $ctrl.pricings[0].interval }),
+                }"
+                data-ng-if="$ctrl.pricings[0].interval !== 0"
+            ></span>
+            <ovh-manager-order-catalog-price
```

Comment:
The component `<ovh-manager-order-catalog-price>` is located into the
Web application (`@ovh-ux/manager-web`).

In my opinion the `wucOrder` component shouldn't live into the _deprecated_
`ngOvhWebUniverseComponents` module.

I suppose that you've dedicated to put it here in order to be able to be shared
across applications but I think we might have to create a dedicated module where
we can rely on the Cart/Order management (Maybe it's already that you have in
mind).

Feel free to tell me If you are not agree.

Reply:
Hello @antleblanc,

> In my opinion the wucOrder component shouldn't live into the deprecated
ngOvhWebUniverseComponents module.

> I suppose that you've dedicated to put it here in order to be able to be shared
across applications but I think we might have to create a dedicated module where
we can rely on the Cart/Order management (Maybe it's already that you have in
mind).


In fact, I'm currently making on an evolution of this component, to handle the case existing in this PR (workflow using /order/cart calls), and a new one (workflow using /services/detach).
So, IMO, this component shouldn't be located in a whole module that concerns only Cart/Order management, as the /services calls won't be in their right place. For the moment, it will still live into `wuc`.

But I agree that a Cart/Order module should exists, that would takes the current `wucOrderCartService`, with `<ovh-manager-order-catalog-price>`. I keep this task in mind, as @Jisay encounters some issue using `wucOrderCartService` in `@ovh-ux/manager-pci`, in a previous PR (https://github.com/ovh/manager/pull/1948)

Let me know your point of view